spec/hbw/features/bp_form/dynamic_conditions_by_select_table_spec.rb

```diff
@@ -32,6 +32,16 @@
     expect(page).to have_content 'Dependent file_upload'
     expect(page).to have_content 'Dependent radio_button'
 
+    in_group_with_label('Dependent submit_select group') do |group|
```

Comment:
I think, it will be enough to test submit_select dynamic conditions only in this test.
I think, it is redundant to copy-past this code to all tests of dynamic conditions.